@navikt/core/react/src/form/combobox/Input/Input.tsx

```diff
@@ -132,16 +141,18 @@ const Input = forwardRef<HTMLInputElement, InputProps>(
           if (activeDecendantId || value) {
             e.preventDefault();
           }
+        } else if (["ArrowLeft", "ArrowUp", "ArrowRight"].includes(e.key)) {
```

Comment:
Found 2 issues with current implementation:
Combobox now triggers onChange for every ArrowLeft/Right/Up, not only when completing autocomplete.

https://github.com/navikt/aksel/assets/26967723/9e92d6e9-0711-469c-a4dd-e97d03ae02de

Also ArrowUp won't work when using arrowkeys in list, so can only navigate down.
https://github.com/navikt/aksel/assets/26967723/c7c26123-7dc8-4319-b206-d5a5f8ab4903

Reply:
Would this be a sufficient guard-clause?

```js
if(value === searchTerm || value === ""){
  // skip onChange 
}
```

Reply:
Pushed some changes that hopefully fixes this 🤞